holidays/ical.py

```diff
@@ -0,0 +1,234 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+import re
+import uuid
+from datetime import datetime, timedelta, timezone
+from pathlib import Path
+from typing import Union
+
+from holidays.version import __version__
+
+# iCal-specific constants
+CONTENT_LINE_MAX_LENGTH = 75
+CONTENT_LINE_DELIMITER = "\r\n"
+CONTENT_LINE_DELIMITER_BYTES = CONTENT_LINE_DELIMITER.encode()
```

Comment:
```suggestion
```
This constant is not used.

Reply:
It's used here in `tests\test_ical.py`, although we can probably just `CONTENT_LINE_DELIMITER.encode()` directly here.
```python
    def _assert_byte_output(self, exporter, expected_bytes):
        output = exporter.generate(return_bytes=True)
        self.assertIsInstance(output, bytes)
        self.assertIn(expected_bytes, output.split(CONTENT_LINE_DELIMITER_BYTES))
```

Reply:
I think, `CONTENT_LINE_DELIMITER.encode()` will be optimal. 👍